stable/appmesh-manager/templates/service.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "appmesh-manager.fullname" . }}-metrics-service
+  namespace: {{ .Release.Namespace }}
+  labels:
+    control-plane: controller-manager
+{{ include "appmesh-manager.labels" . | indent 4 }}
+spec:
+  ports:
+   - name: https
+     port: 8443
+     targetPort: https
```

Comment:
There is no port named `https` in the Deployment spec, I think we should drop this service.

Reply:
ok